### DISCUSSION TITLE
gpt-3.5-turbo, gpt-4 no support - new openai api

### DISCUSSION THREAD

**Discussion by dariomaio** [zilliztech/GPTCache#582] [Q&A]
Due to the changes made to the OpenAI api it seems that cache GPT does not work with gpt-3.5-turbo or gpt-4  models.
Instead it works for gpt-3.5-turbo-instruct
Could you check, please?
Thanks in advance
Dario

**Comment by SimFG** [comment]
maybe you use `openai > 1.0`, because the new openai has changed its apis and there is incompatibility of the openai upgrade interface. In addition, because I have been working too much recently, I have no spare time to adapt to the new version of openai.

**Reply by dariomaio** [reply]
Thanks for your reply. Kind regards.
